static/app/views/performance/transactionSummary/transactionEvents/eventsTable.tsx

```diff
@@ -96,6 +97,7 @@ type Props = {
   customColumns?: ('attachments' | 'minidump')[];
   domainViewFilters?: DomainViewFilters;
   excludedTags?: string[];
+  hasStreamlinedUI?: boolean;
```

Comment:
this feels like something we'll still need after GA of streamline? Maybe we can call it applyEnvironment or something. Not sure what makes sense